usagov/vote-gov-drupal
# Feature/vote 874 drupal core module updates

## Jira ticket

[VOTE-874](https://cm-jira.usa.gov/browse/VOTE-874)

## Description

Update Drupal Core, Modules, and PHP Libraries.

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

1. Locally run `lando retune`.

### QA/Test

1. Login to site as an administrator.
3. Visit the Available Updates admin page (/admin/reports/updates) and confirm that Drupal Core and  Modules are highlighted in green to show that they are current.
4. Browse the site and confirm that expected functionality remains.
5. Review Logs (/admin/reports/dblog) and confirm no new errors.
6. Once deployed to Cloud environment...
7. Test that image uploads to S3 storage remains functional.
8. Once an upload has been confirmed, confirm that the image is accessible as well as the resized (image style) versions.
9. Once merged to dev branch...
10. Confirm that the Dependabot Alert is resolved https://github.com/usagov/vote-gov-drupal/security/dependabot/25 - Composer version updated to 2.7.0+

## Checklist for the Developer




## Checklist for the Peer Reviewers
